source/_integrations/peco.markdown

```diff
@@ -31,10 +31,21 @@ This integration is only for PECO customers. To see if your county is supported,
 
 The PECO Outage Counter integration allows you to get the current count of power outages for any county, and the total for the operating region too.
 
-This integration will create 5 entities.
+There are two different modes available:
```

Comment:
As per comment on the core PR as well, these modes do not make a lot of sense IMHO. 

This should either be not different modes and just provide everything all the time, or should be split into different integrations (as there are two different goals involved?)

../Frenck